ethereum/contracts/SubstrateTypes.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.9;
+
+import "./ScaleCodec.sol";
+
+library SubstrateTypes {
+    function MultiAddressWithID(bytes32 account) internal pure returns (bytes memory) {
+        return bytes.concat(bytes1(0x00), account);
+    }
+
+    function H160(address account) internal pure returns (bytes memory) {
+        return abi.encodePacked(account);
+    }
+
+    function None() internal pure returns (bytes memory) {
+        return hex"11";
```

Comment:
This gets used in place of `bytes1(0x00)` in EthAppPallet and ERC20AppPallet. Should it be 0 instead?

```suggestion
        return hex"00";
```